src/clients/SpokePoolClient.ts

```diff
@@ -947,23 +935,13 @@ export class SpokePoolClient extends BaseAbstractClient {
     );
 
     const tStart = Date.now();
-    // Check both V3FundsDeposited and FundsDeposited events to look for a specified depositId.
     const [fromBlock, toBlock] = [searchBounds.low, searchBounds.high];
     const { maxBlockLookBack } = this.eventSearchConfig;
-    const query = (
-      await Promise.all([
-        paginatedEventQuery(
-          this.spokePool,
-          this.spokePool.filters.V3FundsDeposited(null, null, null, null, null, depositId),
-          { fromBlock, toBlock, maxBlockLookBack }
-        ),
-        paginatedEventQuery(
-          this.spokePool,
-          this.spokePool.filters.FundsDeposited(null, null, null, null, null, depositId),
-          { fromBlock, toBlock, maxBlockLookBack }
-        ),
-      ])
-    ).flat();
```

Comment:
fwiw, might have to back this out. If a relayer fills an old V3FundsDeposited event then the deposit would not be found and the fill would not be refunded.

Reply:
That's true. We always have to keep this in even after we deprecate depositDeprecated()....

Reply:
Restored here: https://github.com/across-protocol/sdk/pull/906/commits/1a1e6c90920d493aeaf451f62690eac39b0a098b